src/main/webapp/script.js

```diff
@@ -11,54 +11,113 @@ function getDashboardItems() {
         const entertainmentItemsContainer = $('#entertainmentItemsContainer');
         entertainmentItemsContainer.empty();
 
-        for (entertainmentItem of entertainmentItemsList) {
-          entertainmentItemsContainer.append(
-              createEntertainmentItemListElem(entertainmentItem));
-        }
+        populateItemGrid(entertainmentItemsContainer, entertainmentItemsList);
       })
       .catch((error) => {
-        console.log('failed to fetch entertainment items: ' + error);
+        console.log(
+            'Failed to fetch entertainment items from DashboardServlet: ' +
+            error);
       });
 }
 
 /**
- * Fetches BlobstoreServlet to get upload Url and then display submission form
- * when it becomes available.
+ * Fetches OMDb item information from OMDb API.
  */
-function getEntertainmentItemForm() {
-  fetch('/blobstore-upload?servletRedirectUrl=/item-submission')
-      .then((response) => response.text())
-      .then((imageUploadUrl) => {
-        const entertainmentItemForm = $('#entertainmentItemForm');
-        entertainmentItemForm.removeClass('hidden');
-        entertainmentItemForm.attr('action', imageUploadUrl);
+function getOMDbItem() {
```

Comment:
`getOmdbItem()`

Reply:
I renamed method to getOmdbItem() in last commit.